dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -83,12 +83,27 @@ HRESULT CFSDropTarget::_CopyItems(IShellFolder * pSFFrom, UINT cidl,
     if (!pszSrcList)
         return E_OUTOFMEMORY;
 
+    BOOL bRenameOnCollision = FALSE;
+    if (bCopy)
```

Comment:
It may be worth checking whether you are copying only one item or more. Because this `_CopyItems` seems to cover the case of many files, it would seem so. (Please correct me if I'm wrong.)

Reply:
`SHFileOperationW` should cover multiple source files.

Reply:
Now the tests on multiple files successful:

![after](https://github.com/reactos/reactos/assets/2107452/646144aa-0d0e-43d9-8f0e-16e6a7521c31)